packages/flutter_bloc/CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
-# 7.0.0-dev.1
-
-- **BREAKING**: remove dependency on `package:provider`
-- feat: add optional `listen` to `BlocProvider`
-- feat: add `ListenProviderExtension`
+# 6.1.0
+
+- feat: add optional `listen` to `BlocProvider` and `RepositoryProvider`
```

Comment:
I know these extensions are coming from `provider`, but it might be useful and give some more contexts to devs to add this information here, just in case someone comes here for more info:

```
context.watch<T>(), which makes the widget listen to changes on T
context.read<T>(), which returns T without listening to it
context.select<T, R>(R cb(T value)), which allows a widget to listen to only a small part of T.
```